app/components/link/link.tsx

```diff
@@ -1,7 +1,9 @@
 import React from "react";
 import router from "../../router/router";
 
-export type LinkProps = React.AnchorHTMLAttributes<HTMLAnchorElement>;
+export type LinkProps = {
+  ignorePreviousState?: boolean;
```

Comment:
I think "previous state" might be ambiguous because there are various other pieces of state in the URL, wdyt about `resetFilter` or something like that?

(also nit add a comment)

```suggestion
  /** Prevents existing filter state from being preserved when navigating. */
  resetFilter?: boolean;
```

Reply:
I like `resetFilters` better than `resetFilter`, but otherwise i updated this here and in `router.tsx`

Reply:
I like that better too :+1: